website/content/docs/platform/k8s/vso/sources/vault/index.mdx

```diff
@@ -305,6 +307,80 @@ The Vault Secrets Operator can optionally cache Vault client information such as
 
 The [Encrypted client cache](/vault/docs/platform/k8s/vso/sources/vault/client-cache) guide will walk you through the steps to enable and configure client cache encryption.
 
+## Instant updates <EnterpriseAlert inline="true" />
+
+The Vault Secrets Operator can instantly update Kubernetes Secrets when changes are made in Vault, instead of waiting for the next refresh interval.
+
+**Supported secret types:**
+- VaultStaticSecret (kv-v1, kv-v2)
```

Comment:
Could we link out to the KV v1 and v2 docs here?

Reply:
Added in 91234da112c67ec275344f8377137a87b8c71f24